src/getCommitMeaning.test.ts

```diff
@@ -40,6 +40,17 @@ describe("getCommitMeaning", () => {
 		["chore(deps): release", "release"],
 		["chore(deps): release 1.2.3", "release"],
 		["chore(deps): release v1.2.3", "release"],
+		// Breaking changes are meaningful no matter the type of the commit.
```

Comment:
[Style] Just nitpicking here, I think the comment is explanatory from the test cases.

```suggestion
```